pkg/snapshot/snapshot.go

```diff
@@ -149,19 +155,42 @@ func (s *Snapshotter) scanFullFilesystem() ([]string, []string, error) {
 	)
 	timing.DefaultRun.Stop(timer)
 
+	filesToResolve := make([]string, 0, len(memFs))
+	for file := range memFs {
+		if strings.HasPrefix(file, "/tmp/dir") {
```

Comment:
is this intentional change?

Reply:
Nope, left over cruft from debugging! Thanks for catching